actions/views/channel_sidebar.ts

```diff
@@ -28,3 +31,70 @@ export function setUnreadFilterEnabled(enabled: boolean) {
         enabled,
     };
 }
+
+export function setCategoryOrder(teamId: string, categoryId: string, channelId: string, newIndex: number) {
+    const getChannelsForCategory = makeGetChannelsForCategory();
```

Comment:
This isn't strictly necessary to do here because the memoization when it's called less frequently, but it's totally fine to leave this way.